docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -35,10 +35,10 @@ A software-defined asset includes the following:
 
 ## Relevant APIs
 
-| Name                                  | Description                                                                                                                                                                                                                  |
-| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <PyObject object="asset" decorator /> | A decorator used to define assets.                                                                                                                                                                                           |
-| <PyObject object="SourceAsset" />     | A class that describes an asset, but doesn't define how to compute it. <PyObject object="SourceAsset" />s are used to represent assets that other assets depend on, in settings where they can't be materialized themselves. |
+| Name                                   | Description                                                                                                                             |
+| -------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="asset" decorator />  | A decorator used to define assets.                                                                                                      |
+| <PyObject object="AssetsDefinition" /> | A class that represents a one or more asset definitions, usually backed by a single op that materializes or observes them all together. |
```

Comment:
small, but i think in general we've been avoiding using "op" in the SDA page (erin can confirm/deny) since it's likely one of the first pages new users read and we want to reduce the amount of new terminology. Could probably just say "function" here instead, but not a big deal either way.

Reply:
👍 will switch to function